Dockerfile

```diff
@@ -47,17 +54,23 @@ RUN adduser --system --disabled-login docker \
  && mkdir -p "$BASE_DIR" "$APP_DIR" "$APP_DIR/src/assets" "$APP_DIR/src/media" \
  && chown -R docker:nogroup "$BASE_DIR" "$APP_DIR"
 
-USER docker
+# Install Poetry
```

Comment:
**issue (blocking):** 
Probably need to add `RUN poetry config virtualenvs.create false` so that we wouldn't get the following errors:
```
pycontw_dev  | Run migration
pycontw_dev  | Traceback (most recent call last):
pycontw_dev  |   File "/app/src/manage.py", line 14, in <module>
pycontw_dev  |     from django.core.management import execute_from_command_line
pycontw_dev  | ModuleNotFoundError: No module named 'django'
pycontw_dev  | Create super user
pycontw_dev  | Traceback (most recent call last):
pycontw_dev  |   File "/app/src/manage.py", line 14, in <module>
pycontw_dev  |     from django.core.management import execute_from_command_line
pycontw_dev  | ModuleNotFoundError: No module named 'django'
pycontw_dev  | Super user already created
pycontw_dev  | Compile localized translation
pycontw_dev  | Traceback (most recent call last):
pycontw_dev  |   File "/app/src/manage.py", line 14, in <module>
pycontw_dev  |     from django.core.management import execute_from_command_line
pycontw_dev  | ModuleNotFoundError: No module named 'django'
pycontw_dev  | Traceback (most recent call last):
pycontw_dev  |   File "/app/src/manage.py", line 14, in <module>
pycontw_dev  |     from django.core.management import execute_from_command_line
pycontw_dev  | ModuleNotFoundError: No module named 'django'
```

Reply:
I modified the scripts to have `enter_dev_env.sh` providing a poetry shell to its user.
`docker-entrypoint.sh` has been removed and `enter_dev_env.sh` does not run those commands in `docker-entrypoint.sh` nor launch the dev server automatically now, anticipating users to handle these processes consciously.

Reply:
Got it, that would be okay, thanks.